### DISCUSSION TITLE
MiniDexed on Raspberry Pi 3A+

### DISCUSSION THREAD

**Discussion by CBorowski-dev** [probonopd/MiniDexed#505] [Q&A] [answered]
Hello, does MiniDexed run on a Raspberry Pi 3A+? In contrast to a Raspberry Pi 3B+, this one only has 512MB memory instead of 1024MB. Is there enough memory to use MiniDexed without restrictions? And if there are restrictions... what would they be?

In advance, thank you for your answers.

Best regards
Christopher

**Comment by Banana71** [comment, marked as answer]
I recently got a Raspberry Pi 3Model A+ and it works great with miniDexed 🎉

**Reply by Banana71** [reply]
The USB type A interfaces are missing

**Reply by probonopd** [reply]
Isn't there 1? You could try a USB hub if you need more...

**Reply by Banana71** [reply]
my mistake, I was wrong with the USB interface
